content/_index.md

```diff
@@ -1,16 +1,33 @@
 +++
-title = 'HobbyFarm Documentation'
-
-[[cascade]]
-type = 'blog'
-toc_root = true
-  
-    [cascade._target]
-    path = '/news/**'
+title = 'Welcome to HobbyFarm'
 
 [[cascade]]
 type = 'docs'
 
   [cascade._target]
   path = '/**'
 +++
+
+HobbyFarm is a [cloud-native](https://en.wikipedia.org/wiki/Cloud-native_computing) training tool that runs in a browser.
+
+HobbyFarm enables developers and engineers to execute hands-on training without requiring a powerful laptop or significant cloud resources. This allows trainers to focus on delivering content and education, rather than fixing issues with outdated virtual machine setups.
+
+| Current Release | Release Date | Supported K8s Version |
+| :--- | :--- | :--- |
+| [hobbyfarm-3.0.1](https://github.com/hobbyfarm/hobbyfarm/releases/tag/hobbyfarm-3.0.1) | July 2023 | v1.25.x |
+
+## Origin
+
+[Rancher Labs](https://rancher.com) operates training sessions called [Rodeos](https://www.rancher.com/events?event_type=rodeos), designed to teach developers and engineers about containers, Kubernetes, and Rancher technologies. These events take place at various locations, such as hotels and conferences, consisting of training presentations and hands-on labs.
+
+These labs were conducted using virtual machines running on attendees' computers. This often caused issues due to common problems such as low-end devices, network issues, and insufficient disk space.
```

Comment:
or even that the user had no rights to install anything

Reply:
Updated to reflect permissions statement.